cmd/block-generator/generator/server.go

```diff
@@ -57,6 +57,7 @@ func MakeServerWithMiddleware(configFile string, addr string, blocksMiddleware B
 	mux.HandleFunc("/v2/accounts/", getAccountHandler(gen))
 	mux.HandleFunc("/genesis", getGenesisHandler(gen))
 	mux.HandleFunc("/report", getReportHandler(gen))
+	mux.HandleFunc("/v2/status/wait-for-block-after/", getStatusWaitHandler(gen))
```

Comment:
Fetcher had a two phase import process, it would skip `wait-for-block-after` until the first missing block.

I'm not sure how much of an impact this has to the performance.